package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "akkeris",
-  "version": "2.3.6",
+  "version": "2.4.12",
```

Comment:
You'll need to bump the version to 2.4.13 to be able to publish your changes to npm.